library/src/scala/Selectable.scala

```diff
@@ -49,6 +49,5 @@ object Selectable:
    *  the additional restriction that the signatures of the refinement and
    *  the definition that implements the refinment must match.
    */
-  @experimental
```

Comment:
This should also be dropped in the tests for this trait: https://github.com/lampepfl/dotty/blob/master/tests/pos/i12211.scala

Reply:
Seems so. Dropped.